src/app/page.tsx

```diff
@@ -1,14 +1,24 @@
-import { Typography } from '@mui/material';
+import { Box, Typography } from '@mui/material';
+import DeviceHubIcon from '@mui/icons-material/DeviceHub';
+import CourseForm from '@/components/CourseForm/CourseForm';
+import CourseList from '@/components/CourseList/CourseList';
 
 export default async function HomePage() {
   return (
-    <main
-      style={{
+    <Box
```

Comment:
Mui Box doesn't seem to like SSR async components and gives an error message in the console (Failed prop type: Invalid prop `children` supplied to `ForwardRef(Box)).  
```suggestion
     <div
      style={{
        display: 'flex',
        flexDirection: 'column',
        justifyContent: 'center',
        alignItems: 'center',
        paddingBottom: 20,
      }}
    >
      <DeviceHubIcon fontSize="small" />
      <h2>Add new Course</h2>
      <CourseForm />
      <CourseList />
    </div>
```

Reply:
I got a ``Failed prop type: Invalid prop `children` supplied to `ForwardRef(Box)`, expected a ReactNode`` error in the console. Wrapping the nested children (DeviceHubIcon, Add new Course, CourseForm, CourseList) in a single `<div>` also seemed to fix the error.